### PR TITLE
fix log error and add more trace logs

### PR DESCRIPTION
**Pull request by jinleileiking**
Relates #47

#### Reference issue
Fixes #...

### CODE REVIEW

**Comment by jinleileiking** [comment]
Not ready for merge. I will and more trace logs.

**Comment by jinleileiking** [comment]
@Sean-Der  Ready for merge :) 
@backkem

File: conn.go [hunk 1]
```diff
@@ -377,13 +378,16 @@ func (c *Conn) handleIncomingPacket(buf []byte) error {
...
+		c.log.Trace("<- Alert")
 		if content.alertDescription == alertCloseNotify {
 			return c.Close()
 		}
 		return fmt.Errorf("alert: %v", content)
 	case *changeCipherSpec:
+		c.log.Trace("<- ChangeCipherSpec")
 		c.setRemoteEpoch(c.getRemoteEpoch() + 1)
 	case *applicationData:
+		c.log.Trace("<- ApplicationData")
```
Comment (backkem) [inline conn.go]: I would avoid this trace since this will be an extremely hot code path in more demanding applications.

File: conn.go [hunk 2]
```diff
@@ -377,13 +378,16 @@ func (c *Conn) handleIncomingPacket(buf []byte) error {
 
 	switch content := r.content.(type) {
 	case *alert:
+		c.log.Trace("<- Alert")
```
Comment (backkem) [inline conn.go]: Maybe print the type of alert as well.

File: conn.go (hunk 1 above)
Comment (jinleileiking) [reply conn.go]: I did not know when will go to this path

Comment (jinleileiking) [reply conn.go]: If I do not enable the trace. Are There will be  any performance issue？

**Pull request closed without merging**